mta/entities/applications.py

```diff
@@ -44,7 +43,8 @@ def is_displayed(self):
     def clear_search(self):
         """Clear search"""
         if self.search.value:
-            self.close_search.click()
+            self.search.fill("")
+            self.browser.refresh()
```

Comment:
Is browser refresh necessary ?

Reply:
yes. Only filling input with `" "`  does not show application.
browser refresh helps to regain the old position.